lib/peeps.rb

```diff
@@ -0,0 +1,36 @@
+require 'pg'
+
+class Peep
+
```

Comment:
One potential improvement here would be to move some of the database connection logic into it's own method or, beyond that, out into a separate class. This would remove some duplication and make the codebase easier to maintain... but I appreciate that you might have run out of time!